src/cljc/rems/application_util.cljc

```diff
@@ -11,3 +13,8 @@
   (contains? (or (:possible-commands application) ;; TODO: remove v1 api usage
                  (:application/permissions application))
              :application.command/save-draft))
+
+(defn get-applicant-name [application]
+  (let [attributes (:application/applicant-attributes application)]
+    (when attributes
+      (first (filter identity (map attributes applicant-name-order))))))
```

Comment:
Maybe bring applicant-name-order closer, or inline (it's not too long) to make it easy to see what kind of data is expected to be in the applicant-attributes?

Reply:
Done, it is now next to the function definition.